zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -570,1247 +570,127 @@ pub(crate) fn change_note_spends_succeed<T: ShieldedPoolTester>() {
     )
 }
 
-pub(crate) fn external_address_change_spends_detected_in_restore_from_seed<
```

Comment:
Please fix this third commit (ad30ad6427034adf042ccf666a0ac6d705fea840):
- Commit message is `remove the raw field on get_tx_history as it broke tests` but the raw field is not removed.
- A bunch of test content is deleted, with no visible move.

In order to review this PR, I _need_ the code moves to be in the same commit (so that `git show COMMIT --color-moved=zebra --color-moved-ws=allow-indentation-change` works. Ideally the commit is as much a pure move as possible, with a fixup commit afterwards, but if the fixups are trivial then doing both at once is okayish.